pandas/tseries/tests/test_timeseries.py

```diff
@@ -2292,6 +2292,20 @@ def test_to_datetime_tz_psycopg2(self):
                                     dtype='datetime64[ns, UTC]')
         tm.assert_index_equal(result, expected)
 
+    def test_datetime_bool(self):
+        # GH13176
+        self.assertRaises(ValueError, to_datetime, False)
+        self.assertTrue(to_datetime(False, errors="coerce") is tslib.NaT)
+        self.assertEqual(to_datetime(False, errors="ignore"), False)
+        self.assertRaises(ValueError, to_datetime, True)
```

Comment:
be explict about passing ``errors='raise'``. do raises in a context manager, its easier to read:

```
with self.assertRaises(ValueError):
    to_datetime(True, errors='raise')
```